packages/common/src/commit-sha.utils.ts

```diff
@@ -74,9 +74,6 @@ export const getCommitDate: (
     // Suppress error logging if not in a git repository
     if (error instanceof Error) {
       if (error.message.startsWith("Command failed")) {
-        logger.info(
```

Comment:
You could also lower it to something lower than `info` which won't go to Datadog.